dashboard/config/initializers/devise.rb

```diff
@@ -286,6 +286,22 @@
   # with a log in with facebook button)
   config.omniauth :clever, CDO.dashboard_clever_key, CDO.dashboard_clever_secret, provider_ignores_state: true
 
+  config.omniauth :openid_connect, {
```

Comment:
So this is an `openid_connect` OmniAuth provider but apparently what gets saved in the db as provider is the endpoint-specific name `the_school_project`?  It makes sense why that would be (there are the endpoint-specific params required to make it work) but I am a little unclear on what we call the two different things. It also seems different than our other auth providers like `facebook`, `google_oauth2`, etc. Is the way to think about it that those products have made their own provider that wraps the internals? And that if The School Project published a gem wrapping OpenId Connect, we could just do something like `config.omniauth :the_school_project, key, key` like the others? And instead of having a wrapper gem we just pass config info to `openid_connect` provider?